app/helpers/govspeak_helper.rb

```diff
@@ -21,6 +21,12 @@ def govspeak_with_attachments_to_html(body, attachments = [])
     wrapped_in_govspeak_div(bare_govspeak_with_attachments_to_html(body, attachments))
   end
 
+  def attachments_govspeak(attachments = [])
```

Comment:
Technically, this doesn't have anything to do with govspeak other than it being rendered in a `div.govspeak`. I'm not sure it even belongs in this helper. Perhaps `AttachmentsHelper` would be better, naming it `block_attachments` to distinguish that it's for the block style rather than inline.